VmAgent.Core/Interfaces/SessionHostConfigurationBase.cs

```diff
@@ -68,6 +68,8 @@ public abstract class SessionHostConfigurationBase : ISessionHostConfiguration
 
         private readonly ISystemOperations _systemOperations;
 
+        protected readonly ISessionHostManager _sessionHostManager;
```

Comment:
do we need to add this here? Can't we just add it on SessionHostContainerConfiguration class?

Reply:
https://github.com/PlayFab/MpsAgent/pull/63#discussion_r713327121 same comment!!

Reply:
sorry, not sure I follow - the other comment is about the addition of another constructor. This comment is about the class SessionHostConfigurationBase (which is inherited by SessionHostContainerConfiguration) having an extra field, instead of adding the field directly to the SessionHostContainerConfiguration class. Hope this makes sense, if not let's make a call to go through it